packages/content/src/lib/content-files-list-token.ts

```diff
@@ -17,11 +16,12 @@ export const CONTENT_FILES_LIST_TOKEN = new InjectionToken<ContentFile[]>(
 
       return Object.keys(contentFiles).map((filename) => {
         const attributes = contentFiles[filename];
+        const slug = attributes['slug'];
 
         return {
           filename,
           attributes,
-          slug: encodeURI(getSlug(filename)),
+          slug: encodeURI(slug) ?? getSlug(filename),
```

Comment:
This should fix the unit and e2e tests

```suggestion
          slug: slug ? encodeURI(slug) : encodeURI(getSlug(filename)),
```